composer.json

```diff
@@ -14,6 +23,7 @@
         "psr/log":                           "~1.0",
         "nikic/php-parser":                  "~1.0",
         "phpdocumentor/reflection-docblock": "~2.0",
+        "phpdocumentor/reflectioncommon":    "dev-feature/getName@dev",
```

Comment:
I have just added this to packagist as package [phpdocumentor/reflection-common](https://packagist.org/packages/phpdocumentor/reflection-common); if there are no other things that need to be changed in this PR I will fix it after a merge; otherwise: can you change the composer file to point that package?